extended-statistics.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Introduction to Extended Statistics
+summary: Learn how to use extended statistics to guide the optimizer.
+---
+
+# Introduction to Extended Statistics
+
+The statistics mentioned in the [Introduction to Statistics](/statistics.md) section, including histograms and Count-Min Sketch, are regular statistics. This information is collected each time statistics are collected manually or automatically. Another class of statistics, as opposed to common statistics, is extended statistics, which are only helpful for optimizer estimation in a specific scenario.
```

Comment:
It's better to have a fixed name "regular statistics". Replace "common statistics" with "regular statistics".